cupy/core/core.pyx

```diff
@@ -2124,20 +2125,20 @@ cpdef ndarray array(obj, dtype=None, bint copy=True, str order='K',
             order = 'A'
         a_cpu = numpy.array(obj, dtype=dtype, copy=False, order=order,
                             ndmin=ndmin)
-        order = 'C' if a_cpu.flags.c_contiguous else 'F'
+        order = None if a_cpu.flags.c_contiguous else 'F'
```

Comment:
Is this change needed? It seems the behavior does not change between `'C'` and `None`.

Reply:
`None` means `'C'`. And, checking `None` is faster than  checking `'C'`.

Reply:
I see, thanks!